pmm/pmm2-server-release.groovy

```diff
@@ -171,24 +171,28 @@ pipeline {
                     DOCKER_MID="\$TOP_VER.\$MID_VER"
                     sg docker -c "
                         set -ex
+                        # push pmm-server
                         docker pull \${DOCKER_VERSION}
+                        docker tag \${DOCKER_VERSION} percona/pmm-server:latest
+                        docker push percona/pmm-server:latest
+                        docker tag \${DOCKER_VERSION} perconalab/pmm-server:latest
+                        docker push perconalab/pmm-server:latest
+
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${VERSION}
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${DOCKER_MID}
                         docker tag \${DOCKER_VERSION} percona/pmm-server:\${TOP_VER}
                         docker push percona/pmm-server:\${VERSION}
                         docker push percona/pmm-server:\${DOCKER_MID}
                         docker push percona/pmm-server:\${TOP_VER}
-                        if [ \${TOP_VER} = 1 ]; then
-                            docker push percona/pmm-server:latest
-                        fi
+
                         docker save percona/pmm-server:\${VERSION} | xz > pmm-server-\${VERSION}.docker
 
+                        # push pmm-client
                         docker pull \${DOCKER_CLIENT_VERSION}
-
-                        if [ \${TOP_VER} = 1 ]; then
-                            docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                            docker push perconalab/pmm-client:latest
-                        fi
+                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
+                        docker push perconalab/pmm-client:latest
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:latest
+                        docker push percona/pmm-client:latest
```

Comment:
In the server above, we tag percona/ first, then perconalab/. Probably the order should be the same for consistency

Reply:
Done!